src/main/java/org/elasticsearch/action/ActionWriteResponse.java

```diff
@@ -156,6 +158,17 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             return builder;
         }
 
+        @Override
+        public String toString() {
+            try {
+                XContentBuilder builder = JsonXContent.contentBuilder();
+                toXContent(builder, new MapParams(Collections.<String, String>emptyMap()));
```

Comment:
I really wonder if we should have `Strings.toString(ToXcontent x)` I think we have this code in a lot of places